docker/entrypoint.sh

```diff
@@ -79,6 +79,11 @@ then
     ETCD_CLUSTER="127.0.0.1:4001"
 fi
 
+mkdir -p ~postgres/.config/patroni
+cat > ~postgres/.config/patroni/patronictl.yaml <<__EOF__
+{dcs_api: 'etcd://${ETCD_CLUSTER}', namespace: /service/}
+__EOF__
+
```

Comment:
I must admit I no longer know what a YAML file looks like ;-)  I've opened the latest spec for v1.2 and I must say these guys are totally nuts.